tests/actor/test_channel_registry.py

```diff
@@ -3,9 +3,12 @@
 
 """Tests for the ChannelRegistry."""
 
+import pytest
+
 from frequenz.sdk.actor import ChannelRegistry
 
 
+@pytest.mark.asyncio
```

Comment:
I think we already get these because we have them configured:  https://github.com/frequenz-floss/frequenz-sdk-python/blob/6ab8228f1227ac6a7742d26045135e1b38419cf8/pyproject.toml#L100

https://pytest-asyncio.readthedocs.io/en/latest/concepts.html#auto-mode

Reply:
True, thanks!
Commit dropped.